pkg/cmd/server/origin/rest/storage_options.go

```diff
@@ -45,6 +46,35 @@ func StorageOptions(options configapi.MasterConfig) restoptions.Getter {
 			{Resource: "netnamespaces"}:                                        "registry/sdnnetnamespaces",
 			{Resource: "netnamespaces", Group: "network.openshift.io"}:         "registry/sdnnetnamespaces",
 		},
+		// storage versions:
```

Comment:
These storage versions look off.  We need to store under `v1` to handle rolling upgrades, right?